test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/CertificateChainUtilityTests.cs

```diff
@@ -127,11 +131,11 @@ public void GetCertificateChain_WithUntrustedSelfIssuedCertificate_ReturnsChain(
                 }
 
                 Assert.Equal(0, logger.Errors);
-                Assert.Equal(RuntimeEnvironmentHelper.IsWindows ? 1 : 2, logger.Warnings);
+                Assert.Equal(RuntimeEnvironmentHelper.IsLinux ? 2 : 1, logger.Warnings);
```

Comment:
what's the extra linux warning?

Reply:
Since the certificate is self-issued, Linux also includes `"unable to get certificate CRL"`, which is a valid warning message to give (since self issued certificates are usually not part of a CRL)